fishcore/ClientApp/src/App.js

```diff
@@ -2,23 +2,19 @@ import React, { Component } from 'react';
 import { Route } from 'react-router';
 import { Layout } from './components/Layout';
 import { Home } from './components/Home';
-import { FetchData } from './components/FetchData';
-import { Counter } from './components/Counter';
+import { AboutData } from './components/FetchAbout';
 
 import './custom.css'
-import DevelopmentWindow from './components/DevelopmentWindow';
 
 export default class App extends Component {
-  static displayName = App.name;
+    static displayName = App.name;
 
-  render () {
-    return (
-      <Layout>
-        <Route exact path='/' component={Home} />
-        <Route path='/counter' component={Counter} />
-        <Route path='/fetch-data' component={FetchData} />
-        <DevelopmentWindow />
-      </Layout>
-    );
-  }
+    render() {
+        return (
+            <Layout>
+                <Route exact path='/' component={Home} />
```

Comment:
Updated the nav bar, removed starter project components.

Reply:
Awesome! Nice and clean.